packages/console/src/components/Switch/index.tsx

```diff
@@ -11,7 +11,7 @@ const Switch = ({ label, ...rest }: Props, ref?: Ref<HTMLInputElement>) => {
     <div className={styles.wrapper}>
       <div className={styles.label}>{label}</div>
       <label className={styles.switch}>
-        <input type="checkbox" {...rest} ref={ref} />
```

Comment:
Will this work? I thought `defaultChecked` property was readonly

Reply:
I tested and it works. If the form is not set any value (not dirty) and submitted, the checkbox value is `false`, but before this commit, it is an empty string `""`.